client/overview/survey/emoticons.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const Emoticons = ( props ) => {
+	const { rating, setReviewRating, disabled, currentRating } = props;
+
+	const buttonCss =
+		'components-button has-icon' +
+		( rating === currentRating ? ' selected' : '' );
+
+	const getIcon = function () {
+		if ( rating === '1' ) {
```

Comment:
Considering the significant markup duplication in that function, I propose passing the unicode emoji text as a prop and retain only one instance of `<span role="img" aria-label="emoticon">`.

Reply:
changed in https://github.com/Automattic/woocommerce-payments/commit/2f32d7cf7f9d001384c5c61315e2c249ac863406 thx!